src/SFA.DAS.ApplyService.Web/Views/Application/Submitted.cshtml

```diff
@@ -11,8 +11,19 @@
             <div class="govuk-panel govuk-panel--confirmation">
                 <h1 class="govuk-panel__title">Application submitted</h1>
                 <div class="govuk-panel__body">
-                    Reference number
-                    <br /><strong>@Model.ReferenceNumber</strong>
+                    <div class="govuk-!-margin-bottom-4">
+                        <span class="govuk-!-font-size-24">Standard applied for</span>
+                        <br />
+                        @if (!string.IsNullOrEmpty(Model.StandardName))
```

Comment:
Should this 'if' wrap the containing DIV?  this is a check the standard name is actually present

Reply:
Yes. Yes it should! 👍